Patches/WarCasket Gundam Addon/Melee_Warcaskets.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+    	<li>WarCasket Gundam Addon</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+			<operations>			
+
+			<!-- Barbatos Katana (Warcasket Broadsword) -->	
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/VFEP_BaseMeleeWeapon_Warcasket[defName="VFEP_WarcasketMeleeWeapon_GKatana"]/tools</xpath>
```

Comment:
Rather than `VFEP_BaseMeleeWeapon_Warcasket`, it should be `ThingDef`, in all occurrences of this.

Reply:
Fixed, I didn't notice the def was different, glad you caught it.